<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameSettings</class>
 <widget class="QDialog" name="GameSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>100</number>
     </property>
     <property name="rightMargin">
      <number>100</number>
     </property>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 700 italic 11pt &quot;Ubuntu Mono&quot;;</string>
       </property>
       <property name="text">
        <string>Set life points</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="SetLifepoints">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(246, 211, 45);</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 700 italic 11pt &quot;Ubuntu Mono&quot;;</string>
       </property>
       <property name="text">
        <string>Set time per move</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="SetTimePerMove">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(246, 211, 45);</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 700 italic 11pt &quot;Ubuntu Mono&quot;;</string>
       </property>
       <property name="text">
        <string>Set initial number of cards</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="SetInitialNumberOfCards">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(246, 211, 45);</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(246, 211, 45);</string>
       </property>
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="leaveButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(246, 211, 45);</string>
       </property>
       <property name="text">
        <string>Leave</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(246, 211, 45);</string>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
